sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -569,3 +572,54 @@ async def upsert_entity(
             return _trim_service_metadata(metadata)
         except HttpResponseError as error:
             _process_table_error(error)
+
+    @distributed_trace
+    def create_batch(
+        self, **kwargs: Dict[str, Any]
+    ) -> TableBatchOperations:
+        """Update/Merge or Insert entity into table.
+
+        return: Table batch operation for inserting new operations
+        rtype: ~azure.data.tables.TableBatchOperations
```

Comment:
Missing first ":"
Also - the comment "Update/Merge or Insert entity into table" is not quite right